fala/apps/category_search/tests/test_search_view.py

```diff
@@ -5,7 +5,7 @@
 from django.conf import settings
 
 
-class SingleCategorySearchViewTests(TestCase):
+class SearchViewTests(TestCase):
```

Comment:
Question 1: did you mean to leave the comment in at line 1?:

`# single_category_search.py`

Question 2: why is this test name singular and this name plural `fala/apps/adviser/tests/test_search_views.py`?

Question 3: Do we need to update the `docs/virtual-env.md`?

Reply:
ooh good spots! Let me correct those things you spotted :)